apps/rush-lib/src/cli/actions/CheckAction.ts

```diff
@@ -19,7 +19,8 @@ export default class CheckAction extends BaseRushAction {
         'version throughout the repository.',
       documentation: 'Checks each project\'s package.json files and ensures that all dependencies are of the ' +
         'same version throughout the repository.',
-      safeForSimultaneousRushProcesses: true
+      safeForSimultaneousRushProcesses: true,
+      rushConfiguration: parser.rushConfiguration
```

Comment:
>rushConfiguration: parser.rushConfiguration [](start = 6, length = 43)

perhaps we should just pass `parser` up to the BaseRushAction, and make `_parser` into a protected getter `parser`?  Seems like most of the actions have a `_parser` private field anyway.

Reply:
There aren't really other useful properties of the parser than rushConfiguration, so that'd just force another level of indirection.

---
In reply to: [177927987](https://github.com/Microsoft/web-build-tools/pull/585#discussion_r177927987) [](ancestors = 177927987)

Reply:
Hmm actually now that I look at most actions, that might make more sense.

---
In reply to: [178216771](https://github.com/Microsoft/web-build-tools/pull/585#discussion_r178216771) [](ancestors = 178216771,177927987)